### DISCUSSION TITLE
Mouse pointer next to the button

### DISCUSSION THREAD

**Discussion by Reisener** [flybywiresim/aircraft#7722] [General]
My version is currently v0.9.1 DEV. MSFS latest version.
Hello,
I only fly with the Virtual Cockpit and operate the buttons and switches with the mouse.
I have to place the mouse pointer very precisely to be able to operate them.
For some switches like autopilot one even to the left of the button. 
In the fire test below the button, otherwise it doesn't work. 
It is very easy to find out, because with the right position you can see an arrow.
Move the virtual screen from right to left and look at the arrow positions on the buttons. You can see this immediately.
This concerns all panels and also the MCDU keys. 
The further away from the center I am, the further I have to go to the right or left with the pointer.
Is this due to the MSFS 2020 or the FBW?
Can you not make this area a little larger and how?
With kind regards
Bernd Reisener
reisener@t-online.de

Translated with www.DeepL.com/Translator (free version)

**Comment by tracernz** [comment]
Hi Bernd,
Check the "lens correction" setting in MSFS settings is **off**.

Unfortunately it "corrects" the graphical display, but not the mouse pointer when on.

**Comment by tracernz** [comment]
![image](https://user-images.githubusercontent.com/9995998/213279595-404a3528-1235-43b6-aed7-eda04bccc677.png)

**Comment by Reisener** [comment]
Hello,
Thank you thank you thank you.
That was a very good tip and it helped.
I fly the FBW every day and think it's great.
Kind regards
Bernd Reisener
